functions-examples/examples/11-package-imports/source.js

```diff
@@ -0,0 +1,26 @@
+// calculate geometric mean off-chain by a DON then return the result
+// valures provided in args array
+
+// imports
+const { escape } = await import("https://deno.land/std/regexp/mod.ts");
```

Comment:
I don't think these imports will be particularly common or useful to devs, so I think we should instead use the same code for Ethers I wrote: https://chainlink-core.slack.com/archives/C04LUBS8RRT/p1701811917803419?thread_ts=1701189601.504329&cid=C04LUBS8RRT